docs/databases/databases.html

```diff
@@ -102,29 +102,36 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
-        <p class="block-text">Kitura has support for the following SQL database plugins:</p>
+        <h2 class="heading-2">Swift-Kuery</h2>
+        <p><a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery">Swift-Kuery</a> is a pluggable SQL database abstraction layer; this is how Kitura provides support for users wishing to use SQL databases, whether directly or via the ORM (see the section below). The intent of the Kuery library is to unify the APIs to the various relational databases, providing a Swifty yet SQL-like API for users. This allows easy switching between different databases.</p>
+        <p class="block-text">Kitura has support for the following SQL databases:</p>
         <ul>
-          <li><a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL">PostgreSQL</a></li>
-          <li><a target="_blank" href="https://github.com/IBM-Swift/SwiftKueryMySQL">MySQL</a></li>
-          <li><a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-SQLite">SQLite</a></li>
+          <li><a target="_blank" href="https://www.postgresql.org">PostgreSQL</a> using the <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-PostgreSQL">Swift-Kuery-PostgreSQL</a> plugin.</li>
+          <li><a target="_blank" href="https://www.mysql.com">MySQL</a> using the <a target="_blank" href="https://github.com/IBM-Swift/SwiftKueryMySQL">SwiftKueryMySQL</a> plugin.</li>
+          <li><a target="_blank" href="https://www.sqlite.org">SQLite</a> using the <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-SQLite">Swift-Kuery-SQLite</a> plugin.</li>
         </ul>
+        <div class="underline"></div>
+        <h2 class="heading-2">Swift-Kuery-ORM</h2>
+        <p><a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery-ORM">Swift-Kuery-ORM</a> is an Object Relational Mapping library (ORM) built for Swift. The ORM provides APIs which simplify the persistence of model objects, by allowing you to interact with the database directly from your Swift types (structs and classes).</p>
+        <p>The ORM doesn’t require you to directly specify how the data should be represented in the database. You define your Swift type as a class or a struct, extend it to conform to the `Model` protocol (which requires no additional code) and the ORM automatically generates a table schema for you. You can then save, fetch, update and delete directly by calling functions on your Swift type.</p>
+        <p>The ORM is built on top of <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery">Swift-Kuery</a>, which means that it is possible to use Kuery to customize SQL queries made to the database, if the functionality of the ORM is insufficient.  It also means that any database supported by Kuery can be used.</p>
```

Comment:
This line confuses me a bit. I think we need to rephase it a bit. E.g:
"The ORM is built on top of <a target="_blank" href="https://github.com/IBM-Swift/Swift-Kuery">Swift-Kuery</a>, means that it can be used with any database that is supported by Kuery. If the functionality of the ORM is insufficient, you can use Kuery directly for more complex SQL queries."

Reply:
👍